### DISCUSSION TITLE
S3 Compatible

### DISCUSSION THREAD

**Discussion by rsidhaarth** [Eugeny/tabby#6785] [Q&A]
Hi,

Is it possible to add support for S3 compatible storage for sync. I see only Amazon, Wasabi and B2.

**Comment by Eugeny** [comment]
S3 is Amazon.

**Comment by rsidhaarth** [comment]
I am speaking about self-hosted S3 storage, where we can enter our own endpoint,bucket name, access key and secret key.

**Reply by Eugeny** [reply]
Oh - please suggest it to the author then: https://github.com/niceit/tabby-cloud-sync-settings/issues
